privacy-configuration/tests.json

```diff
@@ -626,5 +626,139 @@
                 "exceptPlatforms": []
             }
         ]
+    },
+    "ampFormats": {
+        "name": "A URL is able to be extracted from a google formatted URL.",
+        "desc": "Those tests use config4_reference.json where the trackingLinks feature contains regexes",
+        "referenceConfig": "config4_reference.json",
+        "tests": [
+            {
+                "name": "A URL is able to be extracted from a google formatted URL.",
+                "ampUrl": "https://www.google.com/amp/s/www.example.com/some/article",
+                "expectUrl": "https://www.example.com/some/article",
+                "exceptPlatforms": [
+                    "web-extension",
+                    "safari-extension",
+                    "android-browser",
+                    "macos-browser",
+                    "windows-browser"
+                ]
+            },
+            {
+                "name": "A URL is able to be extracted from a google formatted URL with a different TLD.",
+                "ampUrl": "https://www.google.fr/amp/s/www.example.com/some/article",
```

Comment:
let's add some url garbage here like `?bla=ble&lol=123#test` - do we expect it to be retained or removed? Is it usually encoded in real amp urls? Like `&bla=test`?